.mvn/develocity.xml

```diff
@@ -18,6 +18,9 @@
         </capture>
         <!-- https://docs.gradle.com/enterprise/maven-extension/#manual_access_key_configuration -->
         <backgroundBuildScanUpload>#{env['CI'] == null}</backgroundBuildScanUpload>
+        <tags>
+            <tag>Hibernate Search</tag>
+        </tags>
```

Comment:
I question whether this is useful, since Develocity allows filtering by project already :)

Reply:
🙈 👍🏻 yeah it was just for the looks 😄 (I'll go drop it in the Validator patch too) I've dropped it and updated the bot config since now the extension should add a group id instead of the `hibernate.search` tag.